connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/application/ReplE2ESuite.scala

```diff
@@ -283,10 +283,14 @@ class ReplE2ESuite extends RemoteSparkSession with BeforeAndAfterEach {
   test("REPL class in UDF") {
     val input = """
         |case class MyTestClass(value: Int)
-        |spark.range(2).map(i => MyTestClass(i.toInt)).collect()
+        |spark.range(2).
+        |  map(i => MyTestClass(i.toInt)).
+        |  collect().
+        |  map(mtc => s"MyTestClass(${mtc.value})").
+        |  mkString("[", ", ", "]")
       """.stripMargin
     val output = runCommandsInShell(input)
-    assertContains("Array[MyTestClass] = Array(MyTestClass(0), MyTestClass(1))", output)
+    assertContains("""String = "[MyTestClass(0), MyTestClass(1)]"""", output)
```

Comment:
cc @hvanhovell similar as https://github.com/apache/spark/pull/42485, `REPL class in UDF` also need fix for Scala 2.13 ...

Reply:
Doesn't Scala 2.13 print this as `String = "[MyTestClass(value = 0), MyTestClass(value = 1)]"`?